paasta_tools/api/api.py

```diff
@@ -199,6 +205,16 @@ def setup_paasta_api():
     requests_cache.install_cache("paasta-api", backend="memory", expire_after=5)
 
 
+def setup_clog(scribe_host="169.254.255.254", scribe_port=1463):
+    if clog:
+        clog.config.configure(
+            scribe_host=scribe_host,
+            scribe_port=scribe_port,
+            monk_disable=False,
```

Comment:
I suspect we only want one of these disabled, but I'm not sure what the usage is supposed to be (both are disabled by default). @flavray can you weigh in here?

Reply:
These are bare bones configs from `/nail/srv/configs/clog.yaml` so I think they make sense at defaults. These settings are also used in the oom_logger.

Reply:
It's fine to have them both enabled, only the newer versions of yelp_clog allow scribe to be disabled.
Would you mind also adding `preferred_backend="monk"`, otherwise it'll log things to scribe instead of Monk.